ckanext/dgu/theme/templates/page.html

```diff
@@ -15,18 +15,16 @@
 #}
 {%- block content %}
     {% block maintag %}<div role="main">{% endblock %}
-      <div id="content">
-        {% block main_content %}
```

Comment:
You may be 3 steps ahead of me, because I've only just started looking at this, but I'm not clear why you've moved #content down to later in the file and got rid of {% block main_content %}. Is the latter because it is redundant? If so, then I'd prefer to keep it. I've been keen to keep everything in step with core CKAN's page.html, along with the general principal that the more we diverge with core ckan, the more work we create for ourselves with compatibility of shared stuff, such as extensions, etc. Have you seen the comment starting line 8, which explains exactly how the content block comes from core ckan?